java/src/org/openqa/selenium/chrome/ChromeDriverService.java

```diff
@@ -40,12 +40,6 @@
  */
 public class ChromeDriverService extends DriverService {
 
-  /**
-   * System property that defines the location of the ChromeDriver executable that will be used by
-   * the {@link #createDefaultService() default service}.
-   */
-  public static final String CHROME_DRIVER_EXE_PROPERTY = "webdriver.chrome.driver";
-
```

Comment:
I know the system property is still being used, now in the driver class, but I'd prefer we leave a named static variable to contain it because it helps for code readability. So maybe move the removed statement above to `ChromeDriver`?

Same applies to the rest of the drivers.

Reply:
move it to driver or reference it in Service class? Gets weird with Firefox since the constructor takes a `FirefoxDriverService` not a `GeckoDriverService` argument.

Reply:
My intermediary implementation was to implement a `DriverServiceInfo` Interface that each of the drivers implemented — https://github.com/SeleniumHQ/selenium/pull/11491/commits/2e91ebf5c2185cd9b665621ab8a4ab7b41d1a15e

I removed because I decided it was overkill, but it is probably a better way to do this if you think we need to keep the constants?

Reply:
I meant move it to the driver service class.